packages/block-library/src/heading/edit.js

```diff
@@ -56,8 +59,8 @@ function HeadingEdit( {
 			</InspectorControls>
 			{ InspectorControlsColorPanel }
 			<TextColor>
-				<ColorDetector querySelector='[contenteditable="true"]' />
```

Comment:
This component should no longer be needed as the colours can be calculated in an effect.